### ISSUE TITLE
Validation Set

### ISSUE THREAD

**Issue by agenius5** [google-research/pegasus#158]
Hi, @JingqingZ . I have another query.

With `full=True` not set, `continuous_eval` evaluates on 1000 example of validation set. 

1. Did you use all the examples of validation set to determine best checkpoint or just 1000 example?
2. What beam size did you use for evaluation on validaiton set (1 or same as the one used on test set) ? 

I observed that sometimes checkpoint with best validation set numbers doesn't give best test numbres. Is it because I am using only 1000 exmaples?

**Comment by JingqingZ** [comment]
> Did you use all the examples of validation set to determine best checkpoint or just 1000 example?

max 1000, as far as I remember.

> What beam size did you use for evaluation on validaiton set (1 or same as the one used on test set) ?

Essentially all hyper-parameters including beam size were decided on the validation set, then the best setting was used on the test set.

Best performance on validation does not mean best performance on testing, but they should be close.

**Comment by agenius5** [comment]
> Essentially all hyper-parameters including beam size were decided on the validation set, then the best setting was used on the test set.

Didn't know this. Thanks.

Can you confirm that it is the rougeLsum-F score from the text-metrics file that are reported in paper for RougeL-F measure?

I evaluated on CNN_DM dataset and found rougeL-F score to be very low as (around 31) than what it is reported in paper. So, it must be rougeLsum-F. For AESLC and reddit, rougeL-F and rougeLsum-F were pretty close so I didn't noticed it earlier.

**Comment by JingqingZ** [comment]
There are two flavours of ROUGE-L. Please refer to https://github.com/google-research/google-research/tree/master/rouge#two-flavors-of-rouge-l

> Can you confirm that, it is the rougeLsum-F score from the text-metrics file that is reported in paper for RougeL-F measure?

Yes
